tests/e2e/ProgressCircle.spec.ts

```diff
@@ -0,0 +1,21 @@
+import homePage from "./pages/home.page";
+import playgroundPage from "./pages/playground.page";
+
+describe("ProgressCircle", () => {
+    it("renders with a value", () => {
+        homePage.open();
+        homePage.progressText.waitForVisible();
+
+        expect(homePage.progressText.getText()).toBe("-60");
+    });
+
+    it("sets the progress percentage", () => {
```

Comment:
Test description is not straight forward: recommendation ("updates the progress percentage when progress value is changed")